src/plots/linePlot.ts

```diff
@@ -417,7 +417,7 @@ export class Line<X> extends XYPlot<X, number> {
       }
       const projector = attrToProjector[attribute];
       attrToProjector[attribute] = (data: any[], i: number, dataset: Dataset) =>
-        data.length > 0 ? projector(data[0], i, dataset) : null;
```

Comment:
im not sure this is right. data is an array of arrays. as you can see above we use `data[0]` everywhere to refer to the line data